Game_Manual/Echo.md

```diff
@@ -18,4 +18,47 @@ For many commands, passing the `-h` or `--help` flag will show you a helpful mes
 
 For example, to get help with the `/review` command, type `/review --help`.
 
-[los-repo]: https://github.com/LearnersGuild/los
+It's also the primary means by which we communicate with each other - we don't send email at Learners Guild. Because we have so many people communicating on a public platform, it's important to observe some shared Guidelines:
+
+## Guidelines
+
+These guidelines are to support us in making the most of this tool while minimizing distractions and maximizing learning:
+
+Managing **OUTPUT**: guidance for managing the volume of posts & notifications we generate:
+
+1. **Be discerning about WHO you tag - almost never use an @all notification.** Only use an @all notification if it's imperative that everyone in the channel read the message you have sent - assume they will, and that therefore you are interrupting the attention and focus of every person in that channel. @all is the equivalent of ringing the gong. Would you ring the gong for this message? Differentiate between "I'dlikeeveryone to read this" and "Everyonemustread this". Posts in the \#watercooler channel, for example,neverwarrant an @all notification, as it is an entirely recreational / optional channel.
```

Comment:
Agree with @pllearns. Specifically, the text that needs formatting updates:
```
I'd like everyone to
```
```
Everyone must read this
```
I would also suggest the use of backticks to highlight commands, for example `@all`.